pkg/controllers/state/suite_test.go

```diff
@@ -68,6 +68,14 @@ var nodePool *v1beta1.NodePool
 
 const csiProvider = "fake.csi.provider"
 
+var isSyncedTrueLabels = map[string]string{
+	state.SyncedKey: "true",
+}
+
+var isSyncedFalseLabels = map[string]string{
+	state.SyncedKey: "false",
+}
+
 func TestAPIs(t *testing.T) {
```

Comment:
What's your thought on having separate tests specifically for metrics rather than interspersing metric testing into the other test cases?

Reply:
I thought about this and was initially taking this direction, but then found  at least for the isClusterSynced I was just implementing a subset of the existing tests to get the IsClusterSynced cases to align. It made more sense to include it alongside the existing tests rather than duplicating them.